.env.example

```diff
@@ -5,6 +5,7 @@ APP_KEY=base64:nfTdlWswyvfctT6YNgk92bZaKWs5HfhdcyIXWmJUcuk=
 APP_ENV=local
 
 # Must match compose.yml -> db -> environment
+DB_HOST=db
```

Comment:
Main reason for renaming this file to `.env.example` is to avoid the case where tests break even if normal setup seems to work. For anyone following the basic setup (or taking an educated guess at what default setup requires) the example file will be easy to find and use.

We also avoid the case where users end up with multiple `.env.<name of some env that seems plausible>` files that seem like they should do something, even if they are always ignored (since the default `.env` file is prioritised).